pubspec.yaml

```diff
@@ -13,7 +13,10 @@ dependencies:
   barback: '>=0.14.2 <0.16.0'
   logging: '>=0.9.0 <0.12.0'
   path: '>=0.9.0 <2.0.0'
-  smoke: '>=0.1.0 <0.4.0'
+  #smoke: '>=0.1.0 <0.4.0'
+  test: any
+  polymer: '>=1.0.0'
```

Comment:
Me to but see my comments below. It would be a pain to have to write something like:
```dart
class xyz extends Object with Observable, JsProxy {
 @observable @reflectable String field;
}
```
Just to have a simple model that works both in polymer and observable. Consider when having many such things around